src/helpers/getEnrolledMentees.js

```diff
@@ -20,10 +20,10 @@ exports.getEnrolledMentees = async (sessionId, queryParams, userID) => {
 					'stats',
 					'tags',
 					'configs',
-					'visibility',
```

Comment:
exclusion key change visibility -> mentor_visibility
check if you can add the newly introduced keys here